api/services/eventService.js

```diff
@@ -0,0 +1,84 @@
+import database from "../../db/models";
+
+class eventService {
+  static async getEvents() {
+    try {
+      return await database.event.findAll();
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async addEvent(newEvent) {
+    try {
+      return await database.event.create(newEvent);
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async updateEvent(id, updatedEvent) {
+    try {
+      const eventToUpdate = await database.event.findOne({
+        where: { id: Number(id) },
+      });
+
+      if (eventToUpdate) {
+        await database.event.update(updatedEvent, {
```

Comment:
We could make only one request to the db here and try to make `update` right away, we would need to just properly process the response from it.